modules/oxide/src/less/theme/components/toolbar-button/toolbar-button.less

```
@@ -34,10 +35,12 @@
 
 @toolbar-button-active-background-color: @toolbar-button-enabled-background-color;
 @toolbar-button-active-border: @toolbar-button-enabled-border;
-@toolbar-button-active-box-shadow: @toolbar-button-box-shadow; // set to `0 1px 1px 0 rgba(0, 0, 0, .1) inset` for a small push in effect
+@toolbar-button-active-box-shadow: @toolbar-button-box-shadow;
+/* set to `0 1px 1px 0 rgba(0, 0, 0, .1) inset` for a small push in effect */
```

Comment moved down instead of up
```suggestion
/* set to `0 1px 1px 0 rgba(0, 0, 0, .1) inset` for a small push in effect */
@toolbar-button-active-box-shadow: @toolbar-button-box-shadow;
```